resources/views/manager/profile/about.html.twig

```diff
@@ -138,10 +106,21 @@
                             {% for department in departments %}
                                 {% set deparmentName = department.name %}
                                 {% set chosenName = manager.department.name %}
+                                {% if old('department_id') == department.id %}
```

Comment:
If `old('department_id')` is different from chosenName, you could end up with two options selected!